### PR TITLE
docs: add a FAQ about precision

### PR DESCRIPTION
**Pull request by njzjz**

### CODE REVIEW

File: doc/troubleshooting/precision.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+## Data
+
+### Data units and signs
+
+The unit of training data should follow what is listed in [data section](../data/system.md).
+Usually, the package to calculate the training data has different units from those of the DeePMD-kit.
+It is noted that some software label the energy gradient as forces, instead of the negative energy gradient.
+It is neccessary to check them carefully to avoid inconsistent data.
+
+### SCF coverage and data accuracy
```
Comment (wanghan-iapcm) [inline doc/troubleshooting/precision.md:19]: We may also want to discuss the convergence of dft data with respect to energy cutoff and k-space sampling.

File: doc/troubleshooting/precision.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+If the model performs good on the training data, but has bad accuracy on another data, this means some data space is not covered by the training data.
+It can be validated by evaluting the [model deviation](../test/model-deviation.md) with multiple models.
+If the model deviation of these data is high for some data, try to collect more data using [DP-GEN](../third-party/out-of-deepmd-kit.md#dp-gen).
+
+### Values of data
+
+One should be aware that the errors of some data is also affected by the absolute values of this data.
+Stable structures tend to be more precise than unstable structures because unstable structures may have larger forces.
+This is the expected behavior and does not mean that the precision of the model is low.
```
Comment (wanghan-iapcm) [inline doc/troubleshooting/precision.md:37]: Error will be introduced in the Projector augmented wave (PAW) dft calculations when the atoms are very close due to the overlap of pseudo-potentials.

File: doc/troubleshooting/precision.md (hunk 1 above)
Comment (njzjz) [reply doc/troubleshooting/precision.md:19]: Sorry I am not familiar with the plane wave. Maybe you can revise it by yourself...

File: doc/troubleshooting/precision.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+It is noted that some software label the energy gradient as forces, instead of the negative energy gradient.
+It is neccessary to check them carefully to avoid inconsistent data.
+
+### SCF coverage and data accuracy
+
+The accuracy of models will not exceed the accuracy of training data, so the training data should reach enough accuracy.
+On one hand, SCF should converge to a suitable threshold for all points in the training data.
+Sometimes, QM software may generate unstable outliers, which should be removed.
+On the other hand, the data should be extracted with enough digits and stored with the proper precision.
+Large energies may have low precision when they are stored as the single-precision floating-point format (FP32).
```
Comment (wanghan-iapcm) [inline doc/troubleshooting/precision.md:25]:
```suggestion
Here is a checklist for the accuracy of data:
- SCF should converge to a suitable threshold for all points in the training data.
- The convergence of the energy, force and virial with respect to the energy cutoff and k-spacing sample is checked. 
- Sometimes, QM software may generate unstable outliers, which should be removed.
- The data should be extracted with enough digits and stored with the proper precision. Large energies may have low precision when they are stored as the single-precision floating-point format (FP32).
```

**Comment by njzjz** [comment]
GitHub actions fixed in #2131

**Review by wanghan-iapcm — APPROVED** [review]

**Pull request merged**